src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,43 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private DriverService driverService;
+    private CarService carService;
+
+    @Override
+    public void init(ServletConfig config) throws ServletException {
+        driverService = (DriverService) injector.getInstance(DriverService.class);
+        carService = (CarService) injector.getInstance(CarService.class);
+        super.init(config);
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/create-driver-to-car.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/createDriverToCar.jsp").forward(req, resp);
```
спец символы не стоит использовать в URL